layouts/partials/footer.html

```diff
@@ -0,0 +1,19 @@
+<footer>
+    <ul>
+        <li><a href="{{ "index.html" | relLangURL }}">{{ $.Site.Params.index }}</a></li>
+        <li><a href="{{ "countries" | relLangURL }}">{{ $.Site.Params.countries }}</a></li>
+        <li><a href="{{ "tracking" | relLangURL }}">{{ $.Site.Params.analytics }}</a></li>
+        <li>
+            <a id="github" target="_blank"><img src="/img/GitHub-Mark-Light-64px.png" alt=""/> {{ $.Site.Params.contribute }}</a>
```

Comment:
Theses links won't work anymore after this PR is merged: the point to `https://github.com/openfisca/openfisca.org/tree/master/public`, which will  be the same than `https://github.com/openfisca/openfisca.org/tree/hugo/public` afterwards. 

The script needs to be adapted.